manifest_dev.yml

```diff
@@ -0,0 +1,13 @@
+---
+applications:
+  - name: fecfile-api-proxy
+    instances: 1
+    memory: 256M
+    disk_quota: 700M
+    routes:
+      - route: fecfile-web-api-dev.app.cloud.gov
```

Comment:
Let's make the route fecfile-api-proxy

Reply:
updated this and the other manifest with suggested URIs

Reply:
Since we separated the custom domain work into its own workstream, we'll add the *.fecfile.fec.gov routes with a later PR